src/chrome/browser/chrome_content_browser_client.cc

```diff
@@ -1081,9 +1081,7 @@ content::BrowserMainParts* ChromeContentBrowserClient::CreateBrowserMainParts(
   // Construct additional browser parts. Stages are called in the order in
   // which they are added.
 #if defined(TOOLKIT_VIEWS)
-  // Removed !defined(USE_OZONE) for ozone wayland external port
-  // ozone port for wayland should use ChromeBrowserMainExtraPartsViewsLinux
-#if defined(OS_LINUX) && !defined(OS_CHROMEOS)
+#if defined(OS_LINUX) && !defined(OS_CHROMEOS)  && !defined(USE_OZONE)
```

Comment:
#if defined(OZONE_PLATFORM_WAYLAND_EXTERNAL)  || (defined(OS_LINUX) && !defined(OS_CHROMEOS) && !defined(USE_OZONE)) ? And I believe this must go into a separate patch.

Reply:
and let's separate these two commits as well

Reply:
Updated patch with the separate definition with OZONE_PLATFORM_WAYLAND_EXTERNAL and USE_OZONE and made two patches.